src/utils/RequestMetadata.util.ts

```diff
@@ -0,0 +1,8 @@
+import { Map } from 'google-protobuf';
+import { IRequestMetadata } from '../types/RequestMetadata.type';
+
+export function merge(metadataMap: Map<string, string>, metadata: IRequestMetadata): void {
```

Comment:
Can this be abstracted? Rather than its purpose being for RequestMetadata purely, can we for example have a merge map function in something as `map.util.ts`?

Reply:
Rename to `Map.util.ts`